GEMDAQTestSuite.py

```diff
@@ -679,7 +679,7 @@ def report(self):
         return
 
 if __name__ == "__main__":
-    from qcoptions import parser
+    from utils.qcoptions import parser
```

Comment:
Where does this file live when packaged (does the `import` need to be fully qualified)?

Reply:
Outdated legacy code. Change introduced by sed.

For sure this should have started with gempython.vfatqc.

But there’s no support for this script presently.